src/functions/govNotify.ts

```diff
@@ -36,7 +36,9 @@ const govNotify: Handler = async (event: SQSEvent, context?: Context, callback?:
 
       const notifyPromise = downloadService.getCertificate(s3Object.key)
         .then((notifyPartialParams: any) => {
-          return notifyService.sendNotification(notifyPartialParams);
+          if (!notifyPartialParams.shouldEmailCertificate || notifyPartialParams.shouldEmailCertificate === "true") {
```

Comment:
please add a test for this.

Reply:
Agree, we should try to cover new pieces of logic we have especially if they are critical if possible.

Reply:
added more tests